### PR TITLE
report: Handle custom names used in `sklearn_plots`.

### LINKED ISSUE #370: log_sklearn_plot: custom names break make_report
Examples:

```python
>>> from dvclive import Live
>>> live = Live()
INFO:dvclive:Report file (if generated): /Users/dave/repo/dvclive/report.html
>>> y_true = [0, 0, 1, 1]
>>> y_pred = [1, 0, 1, 0]
>>> y_score = [0.1, 0.4, 0.35, 0.8]
>>> live.log_sklearn_plot("confusion_matrix", y_true, y_pred, name="cm")
>>> live.make_report()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/dave/Code/dvclive/src/dvclive/live.py", line 239, in make_report
    make_report(self)
  File "/Users/dave/Code/dvclive/src/dvclive/report.py", line 105, in make_report
    renderers.extend(get_plot_renderers(plots_path / SKLearnPlot.subfolder))
  File "/Users/dave/Code/dvclive/src/dvclive/report.py", line 68, in get_plot_renderers
    properties = SKLEARN_PLOTS[name].get_properties()
KeyError: 'cm'
```

```python
>>> from dvclive import Live
>>> live = Live()
INFO:dvclive:Report file (if generated): /Users/dave/repo/dvclive/report.html
>>> y_true = [0, 0, 1, 1]
>>> y_pred = [1, 0, 1, 0]
>>> y_score = [0.1, 0.4, 0.35, 0.8]
>>> live.log_sklearn_plot("roc", y_true, y_score, name="roc_curve")
>>> live.make_report()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/dave/Code/dvclive/src/dvclive/live.py", line 239, in make_report
    make_report(self)
  File "/Users/dave/Code/dvclive/src/dvclive/report.py", line 105, in make_report
    renderers.extend(get_plot_renderers(plots_path / SKLearnPlot.subfolder))
  File "/Users/dave/Code/dvclive/src/dvclive/report.py", line 67, in get_plot_renderers
    row["rev"] = "workspace"
TypeError: 'str' object does not support item assignment
```

### PR DESCRIPTION
**Pull request by daavoo**
Closes #370

### CODE REVIEW

**Review by daavoo — COMMENTED** [review]
I missed one of the cases in the issue

**Comment by dberenbaum** [comment]
> I missed one of the cases in the issue

Sorry, I should have made more clear that those were 2 separate bugs.

**Comment by daavoo** [comment]
> > I missed one of the cases in the issue
> 
> Sorry, I should have made more clear that those were 2 separate bugs.

Should be both fixed now in the P.R.

File: src/dvclive/report.py [hunk 1]
```diff
@@ -55,17 +58,33 @@ def get_image_renderers(images_folder):
     return renderers
 
 
-def get_plot_renderers(plots_folder):
+def get_plot_renderers(plots_folder, live):
     renderers = []
     for suffix in SKLearnPlot.suffixes:
         for file in Path(plots_folder).rglob(f"*{suffix}"):
             name = file.stem
```
Comment (dberenbaum) [inline src/dvclive/report.py:65]: For names like `train/roc`, I think it needs to be something like:

```suggestion
            name = str(file.relative_to(plots_folder).with_suffix(""))
```

This works, but maybe there's a cleaner solution.

**Review by dberenbaum — APPROVED** [review]

**Pull request merged**